docs/tables/aws_pricing_product.md

```diff
@@ -0,0 +1,55 @@
+# Table: aws_pricing_product
+
+Returns costs, and offer definition, for the corresponding service.
+
+## Examples
+
+### List pricing offers for on-demand shared EC2 c5.2xlarge without pre-installed software, with Linux OS
+
+```sql
+select
+  * 
```

Comment:
Instead of `*`, can you please select some key columns to show? We generally don't like using `*` because when there are a lot of columns it can be overwhelming (and there are quite a lot in this table), and it also doesn't give users an initial direction on what info to check out/what's important.